lib/promiseSftp.coffee

```diff
@@ -295,12 +295,11 @@ class PromiseSftp
           group: file.attrs.gid
           target: null   # TODO
           sticky: null   # TODO
-    
-    @get = (sourcePath) -> Promise.try () ->
+
+    @get = (sourcePath, options = {}) -> Promise.try () ->
       if restartOffset != null
-        options =
-          start: restartOffset
-          flags: 'r+'
+        options.start = restartOffset
```

Comment:
Thanks for the PR.
Should it not be?

```
options.start = options.start || restartOffset
options.flags = options.flags || 'r+'
```

Reply:
Agreed for `options.start`. 
Is there a use case for force overriding `r+` for a `get()`?

Reply:
Just figured if you want to allow full blow options; then allow it. Then people can experiment.